### PR TITLE
gpgmeh: remove detection for crashy gpgme

### PR DESCRIPTION
**Pull request by muelli**
We used to work around a bug on 32bit platforms which crashed the
process. It's been a while and we may as well assume that the issue has
been fixed for good.

This neatly solves a problem of the gpgme version returning not only
ints, but also strings. We don't handle that case very well.

### CODE REVIEW

File: keysign/gpgmeh.py [hunk 1]
```diff
@@ -512,11 +502,6 @@ def sign_keydata_and_encrypt(keydata, error_cb=None, homedir=None):
                     )
 
                 uid_data = UIDExport(signed_keydata, i)
-                # FIXME: Check whether this bug is resolved and the remove this conditional
-                # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=884900
-                if not crashing_gpgme:
-                    log.debug("Data for uid %d: %r, sigs: %r %r", i, uid, uid.signatures, uid_data)
```
Comment (RyuzakiKK) [inline keysign/gpgmeh.py]: If we remove this debug line it should be definitely fine.

**Comment by RyuzakiKK** [comment]
LGTM

File: keysign/gpgmeh.py (hunk 1 above)
Comment (muelli) [reply keysign/gpgmeh.py]: yeah, I didn't intend to remove that line.

**Comment by muelli** [comment]
FWIW: https://repology.org/project/gpgme/versions lists no serious distribution with gpgme < 1.11.0. So this should be good for merging.

**Pull request merged**